spacy/displacy/render.py

```diff
@@ -314,6 +314,8 @@ def render_ents(
             label = span["label"]
             start = span["start"]
             end = span["end"]
+            kb_id = str(span.get("kb_id") or "")
+            kb_url = str(span.get("kb_url") or "")
```

Comment:
This is an extreme nitpick (sorry) but I'd rewrite this as:
```suggestion
            kb_id = str(span.get("kb_id")) or ""
            kb_url = str(span.get("kb_url")) or ""
```